src/client/containers/layout.js

```diff
@@ -27,18 +27,20 @@
 
 import * as bootstrap from 'react-bootstrap';
 
-import FontAwesome from 'react-fontawesome';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import Footer from './../components/footer';
 import PropTypes from 'prop-types';
 import React from 'react';
 import {genEntityIconHTMLElement} from '../helpers/entity';
+import setupIconLibrary from '../helpers/setupIconLibrary';
 
 
 if (!process.env.SSR) {
 	require('../../client/stylesheets/style.less');
 }
 
 const {Alert, MenuItem, Nav, Navbar, NavItem, NavDropdown} = bootstrap;
+setupIconLibrary();
```

Comment:
I think this initialization will cover all the website pages. Is there any much better place to put this logic?

Reply:
- [x] re-factored the initialization setup